ui/component/or-dashboard-builder/src/widgets/or-map-widget.ts

```diff
@@ -123,11 +123,10 @@ export class OrMapWidgetContent extends LitElement {
     /* ---------- */
 
     render() {
-        const config = JSON.parse(JSON.stringify(this.widget!.widgetConfig)) as MapWidgetConfig;
         this.markers = {};
         return html`
-            <or-map id="miniMap" class="or-map" .zoom="${config.zoom ? config.zoom! : undefined}"
-                    .center="${config.center ? config.center! : undefined}"
+            <or-map id="miniMap" class="or-map" .zoom="${this.widget!.widgetConfig.zoom ? this.widget!.widgetConfig.zoom! : undefined}"
```

Comment:
Bit of chaos here with undefined checks haha
You can simply use the ? after a variable; it means it will use the variable if it exists, and otherwise use `undefined`:
```
.zoom="${this.widget?.widgetConfig?.zoom}" .center="${this.widget?.widgetConfig?.center}"
```
Or doesn't this work?

We should get rid of the optional variables in a future update, but thats me to blame for haha